tests/src/tests.rs

```diff
@@ -72,14 +72,22 @@ fn test() {
         }
     };
 
-    let filtered = tests.len();
-    if filtered == 0 {
+    if ARGS.list {
+        for test in tests.iter() {
+            println!("{test}");
+        }
+        println!("{} selected, {skipped} skipped", tests.len());
```

Comment:
all other output is to stderr, so should this be eprintln? or maybe only the final line stderr, so that you could pipe stdout to get the test names.

Reply:
Yeah, `"{test}"` should be to stdout because this is the _intention_ of `--list`, unlike other options where the prints are for informational purposes. Following this logic, `"{} selected, {skipped} skipped"` should be to stderr, as you suggested. Done.